src/components/Board.js

```diff
@@ -9,12 +9,20 @@ const generateSquareComponents = (squares, onClickCallback) => {
   // squares is a 2D Array, but 
   //  you need to return a 1D array
   //  of square components
+  let oneDArray = [];
 
-}
+  for(let i = 0; i < squares.length; i++)
+  {
+    oneDArray = oneDArray.concat(squares[i]);
+  }
+
+  return oneDArray.map((square) => {
+    return <Square value={square.value} id={square.id} key={square.id} onClickCallback ={ () =>onClickCallback(square)}/>
```

Comment:
this works fine! if we'd like to go a step further and stick with standard formatting, we can put each of these props on its own line for readability:

```suggestion
    return (<Square 
        value={square.value} 
        id={square.id} 
        key={square.id} 
        onClickCallback ={ () =>onClickCallback(square)}
    />)
```